spec/unit/graph/sequential_prioritizer_spec.rb

```diff
@@ -27,6 +27,6 @@
   it "fails to prioritize a key contained in an unknown container" do
     expect do
       priorities.generate_priority_contained_in("unknown", "child 1")
-    end.to raise_error
+    end.to raise_error(NoMethodError, /undefined method/)
```

Comment:
This one is a little odd.. underlying code calls `nil.down` resulting in `undefined method 'down' for nil:NilClass`. Ideally the code would protect against that and raise an ArgumentError when the container doesn't have a priority. But I am reluctant to make that change now. How about we include the `'down' for nil` part of the message, as opposed to some other undefined method?